Makefile

```diff
@@ -78,6 +78,9 @@ cleanup:
 	rm -r $(WORKING_DIR)/bin
 	rm -f provider/cmd/$(PROVIDER)/schema.go
 
+examples:
```

Comment:
Won't this be overwritten by ci-mgmt?

If not, should we make `tfgen` depend on `examples`. That would be consistent with the rest of our providers and ensure we didn't need to know anything extra to build or update this one.

Reply:
It will be overwritten but this is what I think it should look like ultimately. Agreed on making tfgen dependent.
Because this is now a hybrid provider, we will have to add logic to ci-management to add this to the Docker Makefile generation.